src/Interpreters/InterpreterShowTablesQuery.cpp

```diff
@@ -33,6 +34,32 @@ String InterpreterShowTablesQuery::getRewrittenQuery()
     if (query.databases)
         return "SELECT name FROM system.databases";
 
+    /// SHOW CLUSTER/CLUSTERS 
+    if (query.clusters)
+    {
+        std::stringstream rewritten_query;
+	rewritten_query << "SELECT cluster FROM system.clusters";
```

Comment:
Do you mean `SELECT * FROM ...`?

Reply:
No, I mean just show cluster name while using "show clusters", 
eg:
```
dell124 :) show clusters;

SHOW CLUSTERS

┌─cluster───────────────────────────┐
│ test_cluster_two_shards           │
│ test_cluster_two_shards           │
│ test_cluster_two_shards_localhost │
│ test_cluster_two_shards_localhost │
│ test_shard_localhost              │
│ test_shard_localhost[1]           │
│ test_shard_localhost_secure       │
│ test_unavailable_shard            │
│ test_unavailable_shard            │
└───────────────────────────────────┘

9 rows in set. Elapsed: 0.002 sec.
```
Do I need to show all the info about clusters, like this:
```
┌─cluster───────────────────────────┬─shard_num─┬─shard_weight─┬─replica_num─┬─host_name─┬─host_address─┬─port─┬─is_local─┬─user────┬─default_database─┬─errors_count─┬─estimated_recovery_time─┐
│ test_cluster_two_shards           │         1 │            1 │           1 │ 127.0.0.1 │ 127.0.0.1    │ 9000 │        1 │ default │                  │            0 │                       0 │
│ test_cluster_two_shards           │         2 │            1 │           1 │ 127.0.0.2 │ 127.0.0.2    │ 9000 │        0 │ default │                  │            0 │                       0 │
│ test_cluster_two_shards_localhost │         1 │            1 │           1 │ localhost │ ::1          │ 9000 │        1 │ default │                  │            0 │                       0 │
│ test_cluster_two_shards_localhost │         2 │            1 │           1 │ localhost │ ::1          │ 9000 │        1 │ default │                  │            0 │                       0 │
│ test_shard_localhost              │         1 │            1 │           1 │ localhost │ ::1          │ 9000 │        1 │ default │                  │            0 │                       0 │
│ test_shard_localhost[1]           │         1 │            1 │           1 │ localhost │ ::1          │ 9000 │        1 │ default │                  │            0 │                       0 │
│ test_shard_localhost_secure       │         1 │            1 │           1 │ localhost │ ::1          │ 9440 │        0 │ default │                  │            0 │                       0 │
│ test_unavailable_shard            │         1 │            1 │           1 │ localhost │ ::1          │ 9000 │        1 │ default │                  │            0 │                       0 │
│ test_unavailable_shard            │         2 │            1 │           1 │ localhost │ ::1          │    1 │        0 │ default │                  │            0 │                       0 │
└───────────────────────────────────┴───────────┴──────────────┴─────────────┴───────────┴──────────────┴──────┴──────────┴─────────┴──────────────────┴──────────────┴─────────────────────────┘

```

@tavplubix  @alexey-milovidov

Reply:
Ok, let `SHOW CLUSTERS` show just names and `SHOW CLUSTER name` show all the info about the cluster. But then it should be `SELECT DISTINCT cluster FROM ...` to show each name only once in `SHOW CLUSTERS`.

Reply:
OK, I will fix this by adding "DISTINCT".